performance/README.md

```diff
@@ -1,18 +1,69 @@
+
+Given this definition of p-value, we need to explicitly call out the common tendancy to apply _probability inversion_ to our observations. To quote [Dr. Tommaso Dorigo](https://www.science20.com/quantum_diaries_survivor/fundamental_glossary_higgs_broadcast-85365) again, "If your ability on the long jump puts you in the 99.99% percentile, that does not mean that you are a kangaroo, and neither can one infer that the probability that you belong to the human race is 0.01%." Using our previously defined terms, the p-value is _not_ the probability that the null hypothesis _is true_.
+
+This brings us to calculating sigma values. Sigma refers to the standard deviation of a statistical model, which is used as a measurement of how far away an observed value is from the expected value. When we say that we have a "3 sigma result" we are saying that if the null hypothesis is true, this is a particularly unlikely observation. Not that the null hypothesis is true. Exactly how unlikely depends on what the expected values from our research hypothesis are. In the context of performance regression testing, if the null hypothesis is false, we are expecting the results to be _slower_ than the expected value not _slower or faster_. Looking at a normal distrubiton below, we can see that we only care about one _half_ of the distribution: the half where the values are slower than the expected value. This means that when we're calculating the p-value we are not including both sides of the normal distribution.
+
+![normal distibution](./images/normal.svg)
+
+Because of this, the following table describes the significance of each sigma level for our _one-sided_ hypothesis:
+
+| σ   | p-value        | scientific significance |
+| --- | -------------- | ----------------------- |
+| 1 σ | 1 in 6         |                         |
+| 2 σ | 1 in 44        |                         |
+| 3 σ | 1 in 741       |  evidence               |
+| 4 σ | 1 in 31,574    |                         |
+| 5 σ | 1 in 3,486,914 |  discovery              |
+
+When detecting performance regressions that trigger alerts, block PRs, or delay releases we want to be conservative enough that detections are infrequently triggered by noise, but not so conservative as to miss most actual regressions. This test suite uses a 3 sigma standard so that only about 1 in every 700 runs is expected to fail the performance regression test suite due to expected variance in our measurements.
+
+### Concrete Example
+
+The following example data was collected locally on a macbook pro using the same tools included in this repository.
```

Comment:
Wouldn't we want to collect the sample mean + stddev on the "same machine" as what will be running in CI? I know that's technically impossible, since GHA is a cloud service using VMs — but it's still possible to match the same basic architecture and memory characteristics, right? Versus a macbook pro running locally

Reply:
you're totally correct. The macbook numbers are all I have right now so I used those to give concrete numbers to the abstract concepts. Your comment here tells me that I should replace these with github action numbers once I have them though.